### PR TITLE
Improve Tutorial Content (01-Introduction-to-Ibis)

### PR DESCRIPTION
**Pull request by AlenkaF**
Updating the content of the first notebook in the examples.

@anjakefala

### CODE REVIEW

**Comment by gforsyth** [comment]
Hey @AlenkaF !  Thanks for working on this!  I'm assuming from your comment that you and @anjakefala are going to iterate on this a little bit?  If I'm wrong, please ping here and I'm happy to take a look over the changes.

File: tutorial/01-Introduction-to-Ibis.ipynb [hunk 1]
```diff
@@ -1,14 +1,33 @@
 {
```
Comment (jcrist) [inline tutorial/01-Introduction-to-Ibis.ipynb:1]: <p>I think we should drop the link here - users will be running these in binder or locally - in either case a link to the next notebook on GitHub won't do the right thing.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-examples/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Comment by jcrist** [comment]
Overall this looks good! Thanks!

**Comment by anjakefala** [comment]
@gforsyth We might iterate over future ones, but if you think this looks good please feel free to go ahead!

File: tutorial/01-Introduction-to-Ibis.ipynb (hunk 1 above)
Comment (jcrist) [reply tutorial/01-Introduction-to-Ibis.ipynb:1]: oop, just noticed that the link is a relative link, nevermind, this is good as is!

**Review by jcrist — APPROVED** [review]
Thanks, merging!

**Pull request merged**